### PR TITLE
Update issue templates to use contact links instead of dummy issue templates.

### PR DESCRIPTION
**Pull request by KotlinIsland**
### Description

This change only affects the Github project, it makes use of the contact_links feature of the new issue page.

If it's desired I could update the issues to use forms. [here is an example of what it could look like](https://github.com/KotlinIsland/basedmypy/issues/new?assignees=&labels=crash&template=crash.yml)

### CODE REVIEW

**Review by sobolevn — COMMENTED** [review]
Does it render for your fork?
Can you please share a screenshot after this patch?

**Comment by KotlinIsland** [comment]
## This is what this pr will render like:
image

## The issue forms could look something like this:
image

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+contact_links:
+  - about: "You can find information on the mypy website."
+    name: "Mypy website"
+    url: "http://www.mypy-lang.org/"
+  - about: "Please check the docs before filing new issues."
+    name: "Common issues and solutions"
+    url: "https://mypy.readthedocs.io/en/stable/common_issues.html"
+  - about: "Please ask and answer usage questions on the mypy Gitter."
+    name: "Questions or Chat"
+    url: "https://gitter.im/python/typing"
```
Comment (hauntsaninja) [inline .github/ISSUE_TEMPLATE/config.yml:10]:
```suggestion
  - about: "Please check the linked documentation page before filing new issues."
    name: "Common issues and solutions"
    url: "https://mypy.readthedocs.io/en/stable/common_issues.html"
  - about: "Please ask and answer any questions on the mypy Gitter."
    name: "Questions or Chat"
    url: "https://gitter.im/python/typing"
```

**Review by hauntsaninja — APPROVED** [review]
Thank you! This is neat. I removed the link to the website, since I'm not sure that it'll help very many people.
I prefer not using forms (but don't feel strongly in case other maintainers prefer them)

**Pull request merged**